Samurai/Modules/CommonModule.cs

```diff
@@ -4,16 +4,14 @@
 using System.Reflection;
 using System.Text;
 using System.Threading.Tasks;
+using AcidChicken.Samurai.Assets;
+using AcidChicken.Samurai.Components;
```

Comment:
All of the local namespace reference instructions have to be included in the local namespace block.